### ISSUE TITLE
return coordinates of regional segmentations

### ISSUE THREAD

**Issue by kristianeschenburg** [miykael/parcellation_fragmenter#22]
For the restricted parcellation, we should return the mappings of each region to the coordinates of each new sub-region fragment.  We can add this to RegionExtractor probably.

**Comment by miykael** [comment]
Yes. But I think we some how also need to make sure that the label numbering increases for each new subregion, something, like `new_labels + np.max(old_labels)`.
